functionary/ui/templates/partials/task_result.html

```diff
@@ -0,0 +1,28 @@
+<div id="result"
+     class="block ml-4"
+     hx-swap-oob="true"
+     {% if not completed %} hx-get="{% url 'ui:task-results' task.id %}?output={{ get_endpoint }}" {% endif %}>
```

Comment:
This was originally used to make sure the page stayed the same when switching environments, but that doesn't appear to be as useful now. It's just defaulting to display_raw since that's the only way to get into the page to start with.